DonjinKrawler-Client/src/main/java/donjinkrawler/adapter/MediaPlayer.java

```diff
@@ -1,6 +1,6 @@
 package donjinkrawler.adapter;
 
 public interface MediaPlayer {
-    void play(String audioType, String fileName);
+    void play(String audioType, String fileName, boolean repeat);
```

Comment:
"Make sure the code formatting is applied"
Since you changed this then apply the formatting where method definitions are separated by an empty line. Thanks !

Reply:
They are separated by empty line...